Release notes for Red Hat Developer Hub 1.4- Update Release Note Texts (#758)
redhat-developer/red-hat-developers-documentation-rhdh: modules/release-notes/snip-bug-fix-rhidp-2727.adoc

[id="bug-fix-rhidp-2727"]

> = Github issues plugin now supports multiple Github integration hosts

Reviewer: ```suggestion
= GitHub issues plugin supports multiple GitHub integration hosts
```

> Before this update, Github issues plugin used the first Github integration it found for all components. This prevented the plugin from being properly used when multiple Github integration hosts were present.

Reviewer: ```suggestion
Previously, the GitHub issues plugin defaulted to using the first GitHub integration it detected for all components. This behavior made it incompatible with setups involving multiple GitHub integration hosts.
```

> With this update, Github issues plugin now supports multiple Github integration hosts. It uses the well-known entity slug annotation `backstage.io/source-location` or `backstage.io/managed-by-location` to determine the appropriate Github integration for a  component. If no integration matches the slug, the first GitHub integration will be selected, maintaining the previous behavior.

Reviewer: ```suggestion
Now, GitHub issues plugin supports multiple GitHub integration hosts. It uses the well-known entity slug annotation `backstage.io/source-location` or `backstage.io/managed-by-location` to determine the appropriate GitHub integration for a component. If no integration matches the slug, the first GitHub integration is selected, maintaining the previous behavior.
```
